scripts/generator/graphql_generator/csharp/SDK/iOS/SummaryItem.cs.erb

```diff
@@ -0,0 +1,19 @@
+#if UNITY_IPHONE
+namespace <%= namespace %>.SDK.iOS {
+    using System.Collections;
+    using System;
+
+    [Serializable]
+    public class SummaryItem: Serializable {
+        #pragma warning disable 0414
+        public string Label;
```

Comment:
Nit: Are these properties meant to be `readonly`? We could add that specifier to these types of properties to strengthen that.